### PR TITLE
Create monitoring-guide.md

### PR DESCRIPTION
**Pull request by abeatrix**
DRAFT

### CODE REVIEW

File: doc/admin/how-to/monitoring-plan.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Monitoring Guide
+Please visit our [Observability Docs](./observability) for more in-depth information about observability.
+
+## What should I be monitoring?
+Sourcegraph comes with built-in monitoring in the form of [Grafana](./observability/metrics#grafana) using [Prometheus](./observability/metrics#prometheus) for metrics and alerting. Generally, you can access Grafana by visiting `https://<your-sourcegraph-url>/-/debug/grafana` for monitoring purpose.
```
Comment (tsenart) [inline doc/admin/how-to/monitoring-plan.md:5]:
```suggestion
Sourcegraph comes with built-in monitoring in the form of [Grafana](./observability/metrics#grafana) using [Prometheus](./observability/metrics#prometheus) for metrics and alerting. Generally, you can access Grafana by visiting `https://<your-sourcegraph-url>/-/debug/grafana` for monitoring purposes.
```

File: doc/admin/how-to/monitoring-plan.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## What does this <ALERT-MESSAGE> mean?
+See [Alert solutions](https://docs.sourcegraph.com/admin/observability/alert_solutions) to learn about each alert and their possible solutions. 
+
+## What’s the threshold for each resource?
+All resources have provisioning indicators that will fire up alerts when the container uses 80% of the container memory limit and 90% of the specified CPU limit.
+
+## How much resources should I add after receiving alerts about running out of resources?
+You should make the decision based on the metrics from your Grafana Dashboard. 
+
+## What are some of the important alerts that I should beaware of?
```
Comment (tsenart) [inline doc/admin/how-to/monitoring-plan.md:27]:
```suggestion
## What are some of the important alerts that I should be aware of?
```

File: doc/admin/how-to/monitoring-plan.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+All alerts are important, but more important when they are in red as that indicate there are issues occuring continuously that require attention.
```
Comment (tsenart) [inline doc/admin/how-to/monitoring-plan.md:28]: Better to recommend focusing only on critical alerts for now.

File: doc/admin/how-to/monitoring-plan.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## How to set up alerts?
+See our Observability Docs on [setting up alerting](./observability/alerting#setting-up-alerting).
+
+## How to create a custom alert?
+Creating a customer alert is not recommended and currently not supported by Sourcegraph.
```
Comment (tsenart) [inline doc/admin/how-to/monitoring-plan.md:34]:
```suggestion
Creating a customer alert is not recommended and currently not supported by Sourcegraph. However, please provide feedback on the monitoring dashboards and alerts if you find anything could be improved.
```

File: doc/admin/how-to/monitoring-plan.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+# Monitoring Guide
+Please visit our [Observability Docs](./observability) for more in-depth information about observability.
+
+## What should I be monitoring?
+Sourcegraph comes with built-in monitoring in the form of [Grafana](./observability/metrics#grafana) using [Prometheus](./observability/metrics#prometheus) for metrics and alerting. Generally, you can access Grafana by visiting `https://<your-sourcegraph-url>/-/debug/grafana` for monitoring purpose.
+
+## When should I be monitoring?
+Whenever you see an alert on the Grafana dashboard.
```
Comment (tsenart) [inline doc/admin/how-to/monitoring-plan.md:8]: This section is a bit fuzzy. What does the act of monitoring mean? Could this be rephrased as "What should I look at when I receive an alert?"

**Review by tsenart — APPROVED** [review]
LGTM for the current state of things. I'm going to be working with all engineering teams to kickoff observability improvements across the board, so I expect change to happen in this area in the coming months.

See https://docs.google.com/document/d/1KjV9pNkwCwqzV5ugW6Bn0eQDZw2L8lVuWaaf7IvVYgQ/edit#

**Comment by tsenart** [comment]
Can we rename the file from `-plan.md` to `-guide.md` to be consistent with the document's title?

File: doc/admin/how-to/monitoring-plan.md (hunk 3 above)
Comment (abeatrix) [reply doc/admin/how-to/monitoring-plan.md]: Update as suggested!

File: doc/admin/how-to/monitoring-plan.md (hunk 5 above)
Comment (abeatrix) [reply doc/admin/how-to/monitoring-plan.md]: That's a good call. I have updated the section with more information about what to look for when experiencing a system performance issue.

**Comment by abeatrix** [comment]
@tsenart thank you so much for taking the time to review the docs with detailed feedback! I have updated the docs with your suggestions and very much looking forward to your observability strategy plan when it kicks off 😎

File: doc/admin/how-to/monitoring-guide.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+# Monitoring Guide
+Please visit our [Observability Docs](./observability) for more in-depth information about observability.
+
+
+## What should I look at when my instance is having performance issue?
```
Comment (tsenart) [inline doc/admin/how-to/monitoring-guide.md:5]:
```suggestion
## What should I look at when my instance is having performance issues?
```

**Review by tsenart — APPROVED** [review]
Feel free to merge :)

**Pull request merged**